_posts/2018-12-14-Bazel-0.20.0.md

```diff
@@ -0,0 +1,91 @@
+---
+layout: posts
+title: Bazel 0.20
+authors:
+  - dslomov
+---
+
+The Bazel team is happy to announce a new version of Bazel, [Bazel 0.20](https://github.com/bazelbuild/bazel/releases/tag/0.20.0).
+
+With this release, we communicate [breaking changes](https://docs.google.com/document/d/1Dj5PBLmPVg9ZyApm4GobM3y-mDgY3mVaqpRVttOe-ZQ/)
+following our [communication policy](https://docs.google.com/document/d/1q5GGRxKrF_mnwtaPKI487P8OdDRh2nN7jX6U-FXnHL0/).
+
+[Breaking changes in 0.20](https://github.com/bazelbuild/bazel/issues?q=is%3Aissue+label%3Abreaking-change-0.20):
+  - [--incompatible_remove_native_http_archive](https://github.com/bazelbuild/bazel/issues/6570)
```

Comment:
the linebreaks for the list don't seem to be rendered https://blog.bazel.build/2018/12/14/Bazel-0.20.0.html